test_sets/policies_smoke.txt

```diff
@@ -0,0 +1,6 @@
+./test_scripts/Polices/build_options/001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
```

Comment:
@istoimenova, There is no invoke of testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY in 001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
Please check which version are you using